Xamarin/SSW.Rewards/SSW.Rewards/Controls/TitleView.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentView xmlns="http://xamarin.com/schemas/2014/forms" 
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="SSW.Rewards.Controls.TitleView">
+  <ContentView.Content>
+      <StackLayout Orientation="Horizontal">
+            <Image Source="SSWWhite_preview" VerticalOptions="CenterAndExpand" />
+            <Label x:Name="title"
+                   Text="ssssdasdw"
```

Comment:
Should this placeholder text be in here?

Reply:
Good catch